southpark-search/tests/distributed/README.md

```diff
@@ -0,0 +1,646 @@
+## Terraform for setting up cloud infrastructure
+
+Configure AWS provider for Terraform as per README:
+https://github.com/jina-ai/cloud-ops
+
+The terraform script here spins up AWS resources, this is the output of running
+`terraform apply`:
+
+data.aws_iam_policy_document.assume_role_policy: Refreshing state...
+
+An execution plan has been generated and is shown below.
+Resource actions are indicated with the following symbols:
+  + create
+ <= read (data resources)
+
+Terraform will perform the following actions:
+```
```

Comment:
This should go to line 8, in order to get the whole console output formatted, right?